x = 100

try:
    print('Checking The Data')
except Exception as e:
    print(e)
else:
    print('Everything Ran Fine')